### PR TITLE
Add Troubleshooting for known issue to ffmpeg_noise

### PR DESCRIPTION
**Pull request by koenhendriks**
## Proposed change

There is a known issue which can be fixed rather easily. This PR adds troubleshooting for this issue to the documentation of the integration. It keeps the same format for troubleshooting as other integrations have in the documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

Related issues:
- https://github.com/home-assistant/core/issues/56421
- https://github.com/home-assistant/core/issues/49183
- https://github.com/home-assistant/core/issues/7081
- https://github.com/home-assistant/core/issues/42204
- https://github.com/home-assistant-libs/ha-ffmpeg/issues/32

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.

- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @koenhendriks, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20446).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/ffmpeg_noise.markdown [hunk 1]
```diff
@@ -73,3 +73,18 @@ To experiment with values:
 ```bash
 ffmpeg -i YOUR_INPUT -vn -filter:a silencedetect=n=-30dB:d=1 -f null -
 ```
+
+### Troubleshooting
+
+#### Unresponsive after a while
+
+If the noise sensor becomes unresponsive, make sure you have `extra_arguments: -nostats` in the configuration, see [Issue 32](https://github.com/home-assistant-libs/ha-ffmpeg/issues/32#issuecomment-727569153) for details.  
+
```
Comment (frenck) [inline source/_integrations/ffmpeg_noise.markdown]: We should not link to issues. If the issue has content that is important, it should be added to the documentation directly.

```suggestion
If the noise sensor becomes unresponsive, make sure you have `extra_arguments: -nostats` in the configuration.  
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by koenhendriks** [comment]
@frenck I've updated as requested. Could you take another look?

**Review by frenck — APPROVED** [review]
Thanks, @koenhendriks 👍

**Pull request merged**